packages/docusaurus-plugin-content-blog/src/blogUtils.ts

```diff
@@ -100,6 +103,60 @@ function formatBlogPostDate(locale: string, date: Date): string {
   }
 }
 
+type AuthorMap = Record<string, Author>;
+
+async function readAuthorListFile(
+  filePath: string,
+): Promise<AuthorMap | undefined> {
+  const AuthorListSchema = Joi.object<AuthorMap>().pattern(
+    Joi.string(),
+    Joi.object({
+      name: Joi.string(),
```

Comment:
I guess we could make this required.
Technically it's possible to just display an avatar but it doesn't make much sense? 
And it's a new feature so we can be more strict.
If anyone has a good use-case for an avatar-only author we'd allow this again but I'd prefer to require author name for now.

Not doing so would allow to provide empty author objects: weird thing to allow

Reply:
Makes sense, I implemented this because otherwise Joi validation would be really hard

Reply:
We cannot make names required, at least not in the map file, because users can still provide the name through front matter. I prefer to give as much flexibility as possible